Documentation/Storage-Configuration/Ceph-CSI/ceph-csi-drivers.md

```diff
@@ -200,11 +200,18 @@ CSI-Addons supports the following operations:
     * [Creating a ReclaimSpaceCronJob](https://github.com/csi-addons/kubernetes-csi-addons/blob/v0.9.1/docs/reclaimspace.md#reclaimspacecronjob)
     * [Annotating PersistentVolumeClaims](https://github.com/csi-addons/kubernetes-csi-addons/blob/v0.9.1/docs/reclaimspace.md#annotating-perstentvolumeclaims)
     * [Annotating Namespace](https://github.com/csi-addons/kubernetes-csi-addons/blob/v0.9.1/docs/reclaimspace.md#annotating-namespace)
+    * [Annotating StorageClass](https://github.com/csi-addons/kubernetes-csi-addons/blob/v0.9.1/docs/reclaimspace.md#annotating-storageclass)
 * Network Fencing
     * [Creating a NetworkFence](https://github.com/csi-addons/kubernetes-csi-addons/blob/v0.9.1/docs/networkfence.md)
 * Volume Replication
     * [Creating VolumeReplicationClass](https://github.com/csi-addons/kubernetes-csi-addons/blob/v0.9.1/docs/volumereplicationclass.md)
     * [Creating VolumeReplication CR](https://github.com/csi-addons/kubernetes-csi-addons/blob/v0.9.1/docs/volumereplication.md)
+* Key Rotation Job for PV encryption
+    * [Creating EncryptionKeyRotationJob](https://github.com/csi-addons/kubernetes-csi-addons/blob/v0.9.1/docs/encryptionkeyrotation.md#encryptionkeyrotationjob)
```

Comment:
Instead of linking to every topic, I wonder if CSI addons could have a table of contents that we just link to once in our docs? Then we don't risk broken links so much. Having a single link to maintain is just simpler.

I agree it's still nice to list the individual features so they can discover the features easier.

Reply:
@Madhu-1 @Rakshith-R how we like to proceed?
Whether to maintain this at csi-addons or Rook?

Reply:
IMO the current change looks good as it list the individual features to give more visibility at a high level in Rook docs. if the user is interested in any specific things they can read it in the csi-addons repo.

Reply:
Yes, it is nice to have the individual links and we can keep it for now, it may just be a challenge to maintain.